<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteUnit</class>
 <widget class="QWidget" name="DeleteUnit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Unit</string>
  </property>
  <widget class="QLabel" name="label_Heading">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Cashier/Price Display Unit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>182</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>30</number>
    </property>
    <item>
     <widget class="QPushButton" name="deleteDevice">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="50,100,50">
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Device">
      <property name="text">
       <string>Device ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Device"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="error_Device">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>DeleteUnit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
